WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -71,6 +71,8 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
         // Restore a disassociated account prior to fixing tokens.
         AccountService(managedObjectContext: ContextManager.shared.mainContext).restoreDisassociatedAccountIfNecessary()
 
+        NullBlogPropertySanitizer().sanitize()
```

Comment:
I'm a bit concerned about potential race conditions with having this in the launch sequence, decoupled from Core Data initialization. It would still be possible that something else tries to save the context before this gets to do its thing.

It would be interesting to do this as part of `mainContext` initialization.

Reply:
Done in d1fb327d8b30996755ef1360cb659b13298e2e81

Reply:
Hmm, a problem with moving this in the `ContextManager.init` is the track events in [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/c8c932df4d32a7ee54145f91bcb994a0ae3093c6/WordPress/Classes/Services/NullBlogPropertySanitizer.swift#L48-L51) are no longer “tracked”. I don't see them in the log at all. I think we also need to ensure that Tracks is initialized first before `ContextManager`.

Reply:
@shiki I was thinking that we can enqueue the values and track later once Tracks is initialized. I'd prefer to avoid changing any Tracks related code in this PR. What do you think?

Reply:
We could probably also make the `PostCoordinator` usage of `ContextManager` to be `lazy` so it doesn't get initialized before everything else. And then force loading `ContextManager` ourselves in the AppDelegate right after Tracks is initialized. This seems risky though. 

> I was thinking that we can enqueue the values and track later once Tracks is initialized.

How do we do that?

Reply:
Turns out that making `uploadsManager` `lazy` is enough to fix it. 😌 ef807792ff823cf747bfd7200126189aceca803c